qulice-maven-plugin/src/test/java/com/qulice/maven/DefaultMavenEnvironmentTest.java

```diff
@@ -128,4 +128,30 @@ public final void passPathsWithWhitespaces()  throws Exception {
             Matchers.notNullValue()
         );
     }
+
+    /**
+     * DefaultMavenEnvironment can produce empty collection when no matches
+     * with checker.
+     * @throws Exception If something wrong happens inside
+     * @todo #676:30min Ideally integration test required to reproduce NPE.
+     *  It should contains at least one exclude rule in
+     *  configuration.excludes block and dependency (I got it
+     *  locally with org.hibernate:hibernate-annotations:3.5.6-Final.
+     *  I suppose, dependency must be compile-time scope but it is only my
+     *  suggestions as far as junit with hamcrest didn't work.
```

Comment:
@vkuchyn now I see, thanks. Please just add that it didn't work for JUnit as test-scope dependency, should be more clear.